caffe2/python/ideep/test_ideep_net.py

```diff
@@ -0,0 +1,175 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from caffe2.proto import caffe2_pb2
+from caffe2.python import core, workspace
+from caffe2.python.models.download import downloadFromURLToFile
+import numpy as np
+import argparse
+import time
+import os.path
+import json
+
+class ModelFetcher:
```

Comment:
Shall we lift this part out from two tests?

Reply:
Where you want to put it? caffe2/python/model? :)